test/e2e/proza-message.test.js

```diff
@@ -58,44 +58,53 @@ describe('vl-proza-message', async () => {
         await message.append('break');
         await message.confirm();
         await assert.eventually.include(message.getText(), '\n');
-        await _resetMessage(message);
+        await resetMessage(message);
     });
 
-    it('als gebruiker kan ik bold stijl toevoegen aan de tekst door tekst te selecteren en dan de stijl knop te gebruiken', async () => {
+    it('als gebruiker kan ik de wysiwyg activeren door alle tekst te selecteren', async () => {
         const message = await vlProzaMessagePage.getMessageFirstDemo();
-        await _enableWysiwyg(message);
+        await message.edit();
+        await message.selectAllText();
+        await vlProzaMessagePage.waitUntilWysiwygOfMessageFirstDemoIsPresent();
+        await assert.eventually.isTrue(message.isWysiwygPresent());
+        await message.confirm();
+        await resetMessage(message);
+    });
+
+    it('als gebruiker kan ik bold stijl toevoegen door in de wysiwyg de stijl knop te gebruiken', async () => {
+        const message = await vlProzaMessagePage.getMessageFirstDemo();
+        await enableWysiwyg(message);
         await assert.eventually.isFalse(message.hasBoldStyle());
         await vlProzaMessagePage.clickWysiwygBoldButton();
         await message.confirm();
         await assert.eventually.isTrue(message.hasBoldStyle());
-        await _resetMessage(message);
+        await resetMessage(message);
     });
 
-    it('als gebruiker kan ik italic stijl toevoegen aan de tekst door tekst te selecteren en dan de stijl knop te gebruiken', async () => {
+    it('als gebruiker kan ik italic stijl toevoegen door in de wysiwyg de stijl knop te gebruiken', async () => {
         const message = await vlProzaMessagePage.getMessageFirstDemo();
-        await _enableWysiwyg(message);
+        await enableWysiwyg(message);
         await assert.eventually.isFalse(message.hasItalicStyle());
         await vlProzaMessagePage.clickWysiwygItalicButton();
         await message.confirm();
         await assert.eventually.isTrue(message.hasItalicStyle());
-        await _resetMessage(message);
+        await resetMessage(message);
     });
 
-    it('als gebruiker kan ik underline stijl toevoegen aan de tekst door tekst te selecteren en dan de stijl knop te gebruiken', async () => {
+    it('als gebruiker kan ik underline stijl toevoegen door in de wysiwyg de stijl knop te gebruiken', async () => {
         const message = await vlProzaMessagePage.getMessageFirstDemo();
-        await _enableWysiwyg(message);
+        await enableWysiwyg(message);
         await assert.eventually.isFalse(message.hasUnderlineStyle());
         await vlProzaMessagePage.clickWysiwygUnderlineButton();
         await message.confirm();
         await assert.eventually.isTrue(message.hasUnderlineStyle());
-        await _resetMessage(message);
+        await resetMessage(message);
```

Comment:
niet beter in de after of before zetten al die "resetMessage", anders falen mogelijks meerdere testen als er 1 faalt?